exercise-src/react-native/14-user-input/02-problem/src/components/FormTextField/FormTextField.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from "react"
+import { useId } from "react"
+import { TextInput } from "react-native"
+import Box from "../../design/Box"
+import { useTheme } from "../../design/theme"
+import Typography from "../../design/Typography"
+
+type Props = {
+  label: string
+  placeholder?: string
```

Comment:
Please remove `placeholder` everywhere. [It’s mostly inaccessible and should very rarely ever be used.]